catalyst_voices/apps/voices/lib/pages/registration/wallet_link/stage/roles_chooser_panel.dart

```diff
@@ -31,13 +29,6 @@ class RolesChooserPanel extends StatelessWidget {
         const Spacer(),
         const RegistrationBackNextNavigation(),
         const SizedBox(height: 10),
```

Comment:
Probably the spacing should be dropped too because it was separating the `RegistrationBackNextNavigation` from `VoicesTextButton`